docs/quick_start/starting.rst

```diff
@@ -18,13 +18,15 @@ For example, to start the Fledge system, open a session to the Fledge device and
 
 If authentication is enabled, which is the default mode for Fledge version 3.0 onward, then a number of the  commands require authentication. Authentication can be accomplished by several means;
 
-  - Set the environment variable *USERNAME* to be the user name.
+  - Set the environment variable *FLEDGE_USER* to be the user name.
```

Comment:
We should revert the change

Reply:
It always was FLEDGE_USER, the previous documentation was wrong.